.coveragerc_appveyor

```diff
@@ -0,0 +1,9 @@
+[run]
+branch = True
+include = C:\projects\clcache\clcache.py
+
+# pytest-cov:
+#   "Note that this plugin controls some options and setting the option in the
+#   config file will have no effect. These include specifying source to be
+#   measured (source option) and all data file handling (data_file and parallel
+#   options)."
```

Comment:
Hm, this file (almost) looks like the existing `.coveragerc` file. Couldn't we modify that to use the full path to `clcache.py` and thus omit all the `--rcfile` and `--cov.config` arguments?